packages/server/src/service/pdfForm/util.ts

```diff
@@ -30,6 +30,23 @@ export const splitFullName = (fullName: string) => {
   return [firstName, middleName, lastName, suffix]
 }
 
+export const stripPhoneNumber = (phoneNumber: string) => {
```

Comment:
This is the kind of logic that's hard to get right.  Can you create a unit test for this and test a bunch of the cases you're hoping to get right?